examples/maps_heatmaps/package.json

```diff
@@ -5,31 +5,37 @@
     "dev": "webpack-dev-server --config ./webpack.dev.js",
     "format": "eslint --fix src/*.ts",
     "lint": "eslint src/*.ts",
-    "test": "jest --passWithNoTests src/*",
-    "watch": "webpack --config ./webpack.dev.js --mode development",
-    "initialize-db": "ts-node-script test/initializeDB.ts"
```

Comment:
this looks like an error. Why remove initialize-db?

Reply:
we fixed it, it was an error